deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -743,6 +743,14 @@ end}.
     {datatype, integer}
 ]}.
 
+{mapping, "default_policies.operator.$id.ha_mode", "rabbit.default_policies.operator", [
```

Comment:
Can we please rename this to "classic_queues.ha_mode"?

Reply:
Adding a suffix to a key in Cuttlefish won't change what it is translated to, if that is what you mean by a "1:1 mapping".

Reply:
Its actually how the rabbit_cuttlefish:aggregate_props handles the prefix. I have a solution imminent though :)